backend/context/config.py

```diff
@@ -61,3 +61,21 @@ def generate(index: int) -> typing.Tuple[LocalAccount, Config]:
         gas_currency_code=testnet.TEST_CURRENCY_CODE,
     )
     return (account, conf)
+
+
+def generate_premainnet(
```

Comment:
couldn't we just keep generate() and use env vars for all the config values?

Reply:
Fixed.